CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## 1.6.0 - 2017-01-29
+## 1.6.0 - 2018-01-29
```

Comment:
I think this is meant to say "Upcoming"?

Reply:
This line was added on 2018-01-29 (ie. previous year).

Reply:
1.6.0 has not been released yet